pyasic/miners/backends/auradine.py

```diff
@@ -193,6 +195,53 @@ async def send_config(self, config: MinerConfig, user_suffix: str = None) -> Non
         for key in conf.keys():
             await self.web.send_command(command=key, **conf[key])
 
+    async def upgrade_firmware(self, *, url: str = None, file_path: str = None, version: str = "latest") -> bool:
```

Comment:
Missing keep_settings here, and file_path can be remove or ignored for now.  Maybe use **kwargs?